src/lib.rs

```diff
@@ -220,9 +220,12 @@ pub trait Events {
 
 impl Cluster {
     pub fn new() -> Cluster {
-        let kubernetes_api_host: &str = &env::var("KUBERNETES_API_HOST").expect("KUBERNETES_API_HOST not set.");
-        let active_host = if kubernetes_api_host.len() > 0 { kubernetes_api_host } else { DEFAULT_HOST };
-        Cluster { host: Url::parse(active_host).unwrap() }
+        let environment_value: &str = &env::var("KUBERNETES_API_HOST").unwrap();
```

Comment:
This call to unwrap() will also panic at runtime. You could try unwrap_or with providing the default kubeproxy url.

Reply:
Alternatively new could take a url as an argument.

Reply:
I like your` unwrap_or` suggestion.  Looks like on anything but an Ok() exception which works for me.

Reply:
Made change